addons/calendar/models/calendar_event.py

```diff
@@ -526,34 +526,38 @@ def get_discuss_videocall_location(self):
     @api.model_create_multi
     def create(self, vals_list):
         # Prevent sending update notification when _inverse_dates is called
-        self = self.with_context(is_calendar_event_new=True)
-        defaults = self.default_get(['activity_ids', 'res_model_id', 'res_id', 'user_id', 'res_model', 'partner_ids'])
+        self = self.with_context(is_calendar_event_new=True, mail_activity_quick_generation=True)
```

Comment:
"mail_activity_skip_notifications" ?

It does a little more than notify (subscribe), but it's globally linked to notifications.